### ISSUE TITLE
User sees an intro message on landing page

### ISSUE THREAD

**Issue by andycochran** [NYCPlanning/labs-regional-viz#20]
The default view of the app should show some introductory text about the app. 

- "Here's what this app does…"
- "Something about People, Jobs, Housing…"
- Should there be a map? If so, what's on it?

We can stub in some filler. 
@daragoldberg to provide final copy.

**Comment by daragoldberg** [comment]
@andycochran Can the intro message be contained **within** the narrative box as the first topic, before prompting user to explore narrative around People, Housing, Jobs, Balance

Default map should be set to county geometry with no data loaded/visualized, boundaries on, county labels on.

**Comment by andycochran** [comment]
Yup. The default app state could just show the first narrative of the first category, and that first narrative is just intro text with a very basic map of the region. 👍

**Comment by allthesignals** [comment]
@andycochran so is this a redirect to the first narrative? And it's just a content question?
